gradle.properties

```diff
@@ -22,12 +22,12 @@ commonsioVersion=2.6
 commonsLang3Version=3.7
 dockerClientVersion=8.16.0
 ecsObjectClientVersion=3.0.6
-spotbugsVersion=4.0.3
-spotbugsAnnotationsVersion=4.0.3
-spotbugsPluginVersion=1.6.9
+spotbugsVersion=4.0.6
+spotbugsAnnotationsVersion=4.0.6
+spotbugsPluginVersion=4.4.4
```

Comment:
LOL. This is a big jump 👍

Reply:
Full disclosure, I basically referenced what Tom did in his `faster-build` branch for these exact versions.  @tkaitchuck Any thoughts?